xAPI-Communication.md

```diff
@@ -768,13 +770,13 @@ Statements are filtered.
 
 <a name="queryLangFiltering" />
 
-###### Language filtering requirements for canonical format statements
+###### Language Filtering Requirements for Canonical Format Statements
 
-* Activity Objects contain Language Map Objects for name, description and interaction components. 
+* Activity Objects contain Language Map Objects within its "name", "description" and various interaction properties. 
```

Comment:
Heh, I wasn't denying they existed, I was trying to be more accurate.  "name" and "description" are properties that directly have these Language Map Objects, they are a bit further down in the interaction components, but I didn't want to say "various properties in interaction components", so I went with just "properties".  I'll revert it and let the readers be smart enough to go look in that section for the usage :)